src/main/java/lessonSevenCinema/Status.java

```diff
@@ -0,0 +1,6 @@
+package lessonSevenCinema;
+
+public enum Status {
```

Comment:
To mogło by być wewnętrzną publiczną statyczną klasą klasy CinemaSeat. W końcu ta klasa nie ma racji bytu bez klasy CinemaSeat.